clusters/sbox/01/kustomization.yaml

```diff
@@ -5,3 +5,5 @@ resources:
 
 patchesStrategicMerge:
 - ../../../apps/flux-system/sbox/01/kustomize.yaml
+- ../../../apps/flux-system/sbox/01/gotk-components_v0_38_3.yaml
+- ../../../apps/flux-system/sbox/01/image-automation-components_v0_38_3.yaml
```

Comment:
image automation should only be deployed to ptlsbox and sbox from what I remember